ddp-workspace/projects/ddp-atcp/src/app/app.module.ts

```diff
@@ -0,0 +1,196 @@
+
+import { TranslateService } from '@ngx-translate/core';
+
+import {
+  DdpModule,
+  LogLevel,
+  ConfigurationService,
+  AnalyticsEventsService
+} from 'ddp-sdk';
+
+import {
+  ToolkitModule,
+  ToolkitConfigurationService
+} from 'toolkit';
+
+import { MatExpansionModule } from '@angular/material/expansion';
+import { MatIconModule } from '@angular/material/icon';
+
+import { WelcomeComponent } from './components/welcome/welcome';
+import { AppComponent } from './components/app/app.component';
+import { FooterComponent } from './components/footer/footer';
+import { AboutUsComponent } from './components/about-us/about-us';
+import { HeaderComponent } from './components/header/header';
+import { MatMenuModule } from '@angular/material';
+import { LanguagesProvider } from './providers/languages.provider';
+import { JoinUsComponent } from './components/join-us/join-us';
+import { AboutInitiativeComponent } from './components/about-initiative/about-initiative';
+import { DataAccessComponent } from './components/data-access/data-access';
+import { PasswordComponent } from './components/password/password';
+import { DashBoardComponent } from './components/dashboard/dashboard';
+import { WorkflowProgressComponent } from './components/workflow-progress/workflow-progress';
+import { ErrorComponent } from './components/error/error';
+import { StatisticsComponent } from './components/statistics/statistics';
+import { ProgressBarComponent } from './components/progress-bar/progress-bar';
+
+const baseElt = document.getElementsByTagName('base');
+
+let base = '';
+if (baseElt) {
+  base = baseElt[0].getAttribute('href');
+}
+
+declare let DDP_ENV: any;
+
+export const tkCfg = new ToolkitConfigurationService();
+tkCfg.studyGuid = DDP_ENV.studyGuid;
+tkCfg.aboutYouGuid = 'ABOUTYOU';
+tkCfg.aboutChildGuid = 'ABOUTCHILD';
+tkCfg.consentGuid = 'CONSENT';
+tkCfg.consentAssentGuid = 'CONSENT_ASSENT';
+tkCfg.parentalConsentGuid = 'PARENTAL_CONSENT';
+tkCfg.releaseGuid = 'RELEASE_SELF';
+tkCfg.releaseMinorGuid = 'RELEASE_MINOR';
+tkCfg.dashboardGuid = 'DASHBOARD';
+tkCfg.lovedOneGuid = 'LOVEDONE';
+tkCfg.lovedOneThankYouGuid = 'THANK_YOU';
+tkCfg.aboutYouUrl = 'about-you';
+tkCfg.aboutChildUrl = 'about-your-child';
+tkCfg.lovedOneUrl = 'loved-one';
+tkCfg.consentUrl = 'consent';
+tkCfg.consentAssentUrl = 'consent-assent';
+tkCfg.parentalConsentUrl = 'parental-consent';
+tkCfg.releaseMinorUrl = 'release-minor-survey';
+tkCfg.releaseUrl = 'release-survey';
+tkCfg.dashboardUrl = 'dashboard';
+tkCfg.doneUrl = 'proxy-thank-you';
+tkCfg.activityUrl = 'activity';
+tkCfg.errorUrl = 'error';
+tkCfg.stayInformedUrl = 'stay-informed';
+tkCfg.lovedOneThankYouUrl = 'loved-one-thank-you';
+tkCfg.infoEmail = 'bistline@broadinstitute.org';
+
+// to configure feed, go to: https://lightwidget.com/widget-info/814feee04df55de38ec37791efea075e
+// need Instagram credentials for @osteosarcomaproject
+tkCfg.lightswitchInstagramWidgetId = '814feee04df55de38ec37791efea075e';
```

Comment:
There are some Osteo artifacts(guids, urls, widget), please, get rid of them too.